client/src/AppRoutes.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+import Home from "./components/Home";
+import SignUp from "./components/SignUp";
+import SignIn from "./components/SignIn";
+
+const AppRoutes = () => {
+  return (
+    //The main routes that can be access through header
+    //and home page has to be added here
```

Comment:
Please change this comment to something like:
`\\ The main routes that will be added to the header must be added here`